docs/community-operator-pipeline.md

```diff
@@ -0,0 +1,40 @@
+# community-operator-pipeline
+
+The community operator pipeline provides a platform for openshift community contributors to validate, share, and distribute openshift operators.
+
+**community-hosted-pipeline**
+
+The community hosted pipeline is used to certify the Operator bundles.
+
+It’s an additional layer of validation that has to run within the Red Hat infrastructure. It is intended to be triggered upon the creation of a bundle pull request and successfully completes with merging it. As part of the community hosted pipeline, the operator submitted in the PR will be tested via preflight trigger. 
```

Comment:
Non-redhat people may not know what preflight or preflight-trigger are. I think we should add a link to help people understand what we are talking about.